### PR TITLE
Fix KillAura & Reach Problem

### PR DESCRIPTION
**Pull request by DanteDEVS**
KillAura Problem:
[18:41:05.686] [Server thread/EMERGENCY]: Crash occurred while handling a packet from session: PlayerName
[18:41:05.687] [Server thread/CRITICAL]: ErrorException: "Undefined variable $player" (EXCEPTION) in "plugins/Zuri/src/Modules/Killaura" at line 132
Reach Problem:
[18:31:16.823] [Server thread/EMERGENCY]: Crash occurred while handling a packet from session: PlayerName
[18:31:16.824] [Server thread/CRITICAL]: ErrorException: "Undefined variable $damager" (EXCEPTION) in "plugins/Zuri/src/Modules/Reach" at line 55

### CODE REVIEW

**Comment by DanteDEVS** [comment]
Tested ✅

**Review by ClydeplayzYT590 — APPROVED** [review]

**Comment by xwertxy** [comment]
Hmm..., my fault...

File: src/Modules/Reach.php [hunk 1]
```diff
@@ -70,7 +70,6 @@ public function reachV1(EntityDamageByEntityEvent $event) : void {
 	// V2 - just check again...
 	public function reachV2(EntityDamageEvent $event) {
 		if ($event instanceof EntityDamageByEntityEvent && $event->getEntity() instanceof Player && $event->getDamager() instanceof Player) {
-			// Use $event->getDamager() instead of $damager
 			if ($this->canBypass($event->getDamager())) {
```
Comment (xwertxy) [inline src/Modules/Reach.php:73]: It is same as it.

**Comment by xwertxy** [comment]
I already fixed this on latest commit.

**Pull request closed without merging**